File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.java
import java.util.*;
class Solution
{
    public int solution(int []A, int []B)
    {
        Arrays.sort(A);
        Arrays.sort(B);
        int sum = 0;
        for(int i = 0, j = A.length - 1; i < A.length; i++,j--){
            sum = sum + A[i]*B[j];
        }
        return sum;
    }
}

File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.py
def solution(A, B):
    answer = 0
    A.sort()  # Sort array A in ascending order
    B.sort(reverse=True)  # Sort array B in descending order
    
    for i in range(len(A)):
        answer += A[i] * B[i]  # Add the product of corresponding elements
        
    return answer